config/development.toml

```diff
@@ -143,7 +143,7 @@ payeezy.base_url = "https://api-cert.payeezy.com/"
 payme.base_url = "https://sandbox.payme.io/"
 paypal.base_url = "https://www.sandbox.paypal.com/"
 payu.base_url = "https://secure.snd.payu.com/"
-powertranz.base_url = "https://staging.ptranz.com/api/spi/"
+powertranz.base_url = "https://staging.ptranz.com/api/"
```

Comment:
Observed that you have modified base url here, can you please modify in other places as well, as the base url there is still "https://staging.ptranz.com/api/spi/"